### PR TITLE
Fix retain credentials info on connect frame

### PR DESCRIPTION
**Pull request by oskarhane**
Stale information was presented on connect frame.
Now it updates when configuration is read from dbms.

### CODE REVIEW

File: src/browser/modules/Stream/Auth/ConnectionForm.test.js [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+  )
+
+  // Then
+  expect(getByText(/my-username/i)).toBeDefined()
+  expect(getByText(/bolt:\/\/my-host/i)).toBeDefined()
+  expect(
+    getByText(/Connection credentials are\sstored in your web browser./i)
+  ).toBeDefined()
+
+  // When not storing vredentials anymore
```
Comment (linuslundahl) [inline src/browser/modules/Stream/Auth/ConnectionForm.test.js]: credentials

File: src/browser/modules/Stream/Auth/ConnectionForm.test.js [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+  // Then form should be there
+  expect(getByText(/connect url/i)).toBeDefined()
+
+  // When connection is ok
+  fireEvent.change(getByTestId('boltaddress'), { target: { value: host } })
+  fireEvent.change(getByTestId('username'), { target: { value: username } })
+  fireEvent.change(getByTestId('password'), { target: { value: 'xxx' } })
+  fireEvent.click(getByTestId('connect'))
+
+  // When faking proagation
```
Comment (linuslundahl) [inline src/browser/modules/Stream/Auth/ConnectionForm.test.js]: propagation?

File: src/browser/modules/Stream/Auth/ConnectionForm.test.js (hunk 1 above)
Comment (oskarhane) [reply src/browser/modules/Stream/Auth/ConnectionForm.test.js]: Fixed before your comment :)

File: src/browser/modules/Stream/Auth/ConnectionForm.test.js (hunk 2 above)
Comment (oskarhane) [reply src/browser/modules/Stream/Auth/ConnectionForm.test.js]: 👍

**Pull request merged**